docs/dm/zh_CN/vendors/aliyun-rds.md

```diff
@@ -0,0 +1,54 @@
+阿里云 RDS 支持限制
+===
+
+### 内容索引
+
+- [implict_primary_key 造成同步中断](#implict_primary_key-造成同步中断)
+- [缺少 SUPER privilege(s) 权限 dump 失败](#缺少-super-privileges-权限-dump-失败)
```

Comment:
Update this link

Reply:
:anguished: already merged.

Reply:
I'll push a commit to the doc branch directly to update it.

Reply:
Ok 🤣